doc/classes/KinematicBody2D.xml

```diff
@@ -31,30 +32,35 @@
 			<argument index="0" name="slide_idx" type="int">
 			</argument>
 			<description>
+				Returns a [KinematicCollision2D], which contains information about a collision that occured during the last [method move_and_slide] call. Since the body can collide several times in a single call to [method move_and_slide], you must specify the index of the collision in the range 0 to ([method get_slide_count]()-1).
 			</description>
 		</method>
 		<method name="get_slide_count" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the number of times the body collided and changed direction during the last call to [method move_and_slide].
 			</description>
 		</method>
 		<method name="is_on_ceiling" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if the body is on the ceiling. This value is only updated when [method move_and_slide] is called.
```

Comment:
This value is only updated -> This value only updates. You can probably replace all instances at once. It's not an incorrect use of passive, but in general direct voice is friendlier for non-native speakers.
Also, what updates: the return value or the internal data? Suggestion for this and other methods:  `Only updates when calling [method move_and_slide].`